### ISSUE TITLE
Active/open class on collection

### ISSUE THREAD

**Issue by GillesVermeulen** [DataTables/Buttons#156]
It would be useful if an extra class was added to a collection button if it was opened. This way open collection buttons can be styled differently.

**Comment by DataTables** [comment]
Good point - I like that idea.

**Comment by AllanJard** [comment]
There is the `aria-expanded` attribute which is either `true` or `false`. That could be used as a selector to determine whether styles should be added or not.
